coq-mathcomp-multinomials.opam

```
@@ -14,7 +14,7 @@ build: [
 depends: [
   "coq"                    {(>= "8.10" & < "8.15~") | = "dev"}
   "dune"                   {>= "2.5"}
-  "coq-mathcomp-ssreflect" {(>= "1.12" & < "1.13~") | = "dev"}
+  "coq-mathcomp-ssreflect" {(>= "1.12" & < "1.14~") | = "dev"}
   "coq-mathcomp-algebra"
   "coq-mathcomp-bigenough" {(>= "1.0" & < "1.1~") | = "dev"}
   "coq-mathcomp-finmap"    {(>= "1.5" & < "1.6~") | = "dev"}
```

I think we can simply remove the upper bound here (and `coq` above). Right? I mean:
```suggestion
  "coq-mathcomp-ssreflect" {>= "1.12"}
  "coq-mathcomp-algebra"
  "coq-mathcomp-bigenough" {>= "1.0"}
  "coq-mathcomp-finmap"    {>= "1.5"}
```